examples/example_material.xomp

```diff
@@ -15,588 +15,6 @@
-          [1210e-9,10.80782605],
-          [1215e-9,10.85926339],
-          [1220e-9,10.9110202],
-          [1225e-9,10.96181208],
-          [1230e-9,11.01266681],
-          [1235e-9,11.06849385],
-          [1240e-9,11.11569439],
-          [1245e-9,11.16984741],
-          [1250e-9,11.21616325],
-          [1255e-9,11.26655345],
-          [1260e-9,11.31619114],
-          [1265e-9,11.36079431],
-          [1270e-9,11.41081797],
-          [1275e-9,11.47288865],
-          [1280e-9,11.51937378],
-          [1285e-9,11.56250415],
-          [1290e-9,11.61775711],
-          [1295e-9,11.66829595],
-          [1300e-9,11.71967858],
-          [1305e-9,11.7737027],
-          [1310e-9,11.82020872],
-          [1315e-9,11.86438552],
-          [1320e-9,11.91930196],
-          [1325e-9,11.96737106],
-          [1330e-9,12.01542787],
-          [1335e-9,12.06389822],
-          [1340e-9,12.1162431],
-          [1345e-9,12.16532652],
-          [1350e-9,12.21775953],
-          [1355e-9,12.26700903],
-          [1430e-9,12.99198306],
-          [1435e-9,13.0402495],
-          [1440e-9,13.08984959],
-          [1445e-9,13.13438949],
-          [1450e-9,13.19350911],
-          [1455e-9,13.23424936],
-          [1460e-9,13.2825952],
-          [1465e-9,13.32973008],
-          [1470e-9,13.37524914],
-          [1475e-9,13.42119816],
-          [1480e-9,13.46516297],
-          [1485e-9,13.51960433],
-          [1490e-9,13.56523813],
-          [1495e-9,13.61515704],
-          [1500e-9,13.66027168],
-          [1505e-9,13.70964938],
-          [1510e-9,13.75760575],
-          [1515e-9,13.81378197],
-          [1520e-9,13.8577198],
-          [1525e-9,13.90548777],
-          [1530e-9,13.94978109],
-          [1535e-9,13.9955611],
-          [1540e-9,14.0463704],
-          [1545e-9,14.08906725],
-          [1550e-9,14.13278052],
-          [1555e-9,14.18605286],
-          [1560e-9,14.2300396],
-          [1565e-9,14.27204602],
-          [1570e-9,14.33214219],
-          [1575e-9,14.37325593],
-          [1580e-9,14.41873986],
-          [1585e-9,14.47146449],
-          [1590e-9,14.51965833],
-          [1595e-9,14.56653756],
-          [1600e-9,14.60865354],
-          [1605e-9,14.65292487],
-          [1610e-9,14.70647543],
-          [1615e-9,14.75635087],
-          [1620e-9,14.82100212],
-          [1625e-9,14.84862799],
-          [1630e-9,14.8956765],
-          [1635e-9,14.94712102],
-          [1640e-9,14.99170484],
-          [1645e-9,15.02392461],
-          [1650e-9,15.08179888],
-          [1655e-9,15.1223348],
-          [1660e-9,15.17582154],
-          [1665e-9,15.23233469],
-          [1670e-9,15.2773787],
-          [1675e-9,15.34740513],
-          [1680e-9,15.38951869],
-          [1685e-9,15.4724516],
-          [1690e-9,15.49621661],
-          [1695e-9,15.53351552],
-          [1700e-9,15.55632073]
-        ]
-      }
-    ]
+    "electromagnetic_properties_uri": "example_material_emp.xomlt"
```

Comment:
electromagnetic_properties_uri don't you meant url ?

Reply:
URI stands for Uniform Resource Identifier. In this case it is a relative path to the property table file.